packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -13,6 +13,7 @@ import 'web_kit_api_impls.dart';
 /// Times at which to inject script content into a webpage.
 ///
 /// Wraps [WKUserScriptInjectionTime](https://developer.apple.com/documentation/webkit/wkuserscriptinjectiontime?language=objc).
+
```

Comment:
Why all the whitespace between comments and the things they are commenting in this file?

Reply:
Accident! I used annotations and a template processor to generate most of the Objective-C code. I assumed the formatter would remove the Dart space once the annotations were removed.